dl/src/main/scala/com/intel/analytics/bigdl/optim/DistriOptimizer.scala

```diff
@@ -246,6 +247,29 @@ object DistriOptimizer {
         logger.debug("Dropped modules: " + (driverSubModelNum - finishedModelNum))
         lossArray = new Array[Double](_subModelNumber)
 
+        // record the hyper parameters to debug log, only support SGD now.
+        optimMethod match {
+          case sgd: SGD[T] =>
+            val hpScheduler = state.getOrElse[HyperParameterScheduler](
```

Comment:
why not just print state? If there's new member in state, you have to change this piece of code.